core/src/client/lwm2m_static.c

```diff
@@ -318,6 +318,93 @@ int AwaStaticClient_Process(AwaStaticClient * client)
     return result;
 }
 
+static AwaLwm2mResult AwaStaticClientDefaultHandler(AwaStaticClient * client, AwaOperation operation, AwaObjectID objectID, 
```

Comment:
Having a single handler for both object and resource operations seems a little clunky to me, is this the intended use of the API?